apps/hash-frontend/src/pages/shared/entity-type-page/type-preview-slide.tsx

```diff
@@ -145,7 +146,10 @@ export const TypePreviewSlide: FunctionComponent<TypePreviewSlideProps> = ({
                     }
                     ontologyChip={
                       <Link
-                        href={remoteEntityType.schema.$id}
+                        href={
+                          generateLinkParameters(remoteEntityType.schema.$id)
+                            .href
+                        }
```

Comment:
```suggestion
                        href={remoteEntityType.schema.$id}
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/3572/commits/b31daf5c29c4daa2fcd9729c030f2852da744120